frontends/web/src/locales/en/app.json

```diff
@@ -189,6 +189,12 @@
     }
   },
   "deviceTampered": "Has your BitBox been supplied with a recovery password? If so, stop the setup process and contact support immediately. Shift will never give you a ready made wallet or make password recommendations.",
+  "deviceinfo": {
```

Comment:
remove, everything here is fed into the translation pipeline, but this is not fixed yet. Let's wait until the real screens are here.